tests/src/Phansible/Controller/BundleControllerTest.php

```diff
@@ -59,14 +59,14 @@ public function testShouldResponseWithErrorMessage()
         $data = [
             'vagrant_local' => [
                 'vm' => [
-                    'ip' => ''
-                ]
-            ]
+                    'ip' => '',
+                ],
+            ],
```

Comment:
This as last element should not have the `,`

Reply:
It's actually ok in theory... the real issue is, why are they there? I don't have them locally 😄